tests/cli/test_compose_qcow2.sh

```diff
@@ -14,7 +14,7 @@ set -e
 
 CLI="${CLI:-./src/bin/composer-cli}"
 QEMU_BIN="/usr/bin/qemu-system-$(uname -m)"
-QEMU="$QEMU_BIN -machine accel=kvm:tcg"
+QEMU="$QEMU_BIN -machine accel=tcg"
```

Comment:
Now that we're disabling the tests, we don't need to switch to software emulation.